bgpd/bgp_debug.c

```diff
@@ -879,6 +879,13 @@ DEFPY (debug_bgp_bestpath_prefix,
        "IPv4 prefix\n"
        "IPv6 prefix\n")
 {
+	struct prefix *argv_p;
+	int idx_ipv4_ipv6_prefixlen = 3;
+
+	argv_p = prefix_new();
+	str2prefix(argv[idx_ipv4_ipv6_prefixlen]->arg, argv_p);
+	apply_mask(argv_p);
```

Comment:
You need to call apply_mask() on the `bestpath` prefix created by clippy. `argv_p` is not being used anywhere and is leaking memory.

Reply:
I'm going to change this DEFPY to DEFUN so this is consistent with the `debug bgp updates prefix` code.  This will allow us to do the needed apply_mask() cleanly.

Reply:
Your call, but since you removed the DEFPY you need to update bgpd/Makefile.am and remove this line:
```
bgp_debug.o: bgp_debug_clippy.c
```

You also need to remove these lines in bgpd/bgp_debug.c:
```
 #ifndef VTYSH_EXTRACT_PL
 #include "bgp_debug_clippy.c"
 #endif
```